File: Algorithms/JAVA/CH0/Factorial.java
public class Factorial{
    public static int factorial(int n){
        if(n<=1) return 1;
        else return factorial(n-1)*n;
    }
    public static void main(String args[]){
        int num = 5;
        num = factorial(num);
        System.out.printf("Result: %d\n",num);
    }
}

File: Algorithms/C#/CH0/Factorial.cs
using System;

namespace Factorial
{
    class Program
    {
        static int Factorial(int n){
            if(n==1) return 1;
            return Factorial(n-1) * n;
        }
        static void Main(string[] args)
        {
            int num = Factorial(5);
            Console.WriteLine("Result: {0}",num);
        }
    }
}
